### ISSUE TITLE
Django `request` tab not following `params_filters`

### ISSUE THREAD

**Issue by justmobilize** [bugsnag/bugsnag-python#317]
### Describe the bug
[Here](https://github.com/bugsnag/bugsnag-python/blob/83a4b9f5529fc769d65f65500c848ac8e743400b/bugsnag/django/__init__.py#L59) the `url` is being built with `build_absolute_uri()` and thus not being updated with `params_filters` like the `GET` data does.

### Steps to reproduce
1. configure Bugsnag with `params_filters`
2. raise an exception from a GET request with a param in params_filters
3. See it's filtered in the GET block
4. See it's not filtered in the request.url

### Environment
* Bugsnag version: `4.2.0`
* Python version: `3.8`
* Integration framework version:
    * ASGI:
    * Celery:
    * Django: 2.2.26
    * Flask:
    * Tornado:
    * WSGI:

### Example Repo 

- [ ] Create a minimal repository that can reproduce the issue
- [ ] Link to it here:

### Example code snippet 

```
# (Insert code sample to reproduce the problem)
```

In django settings setting:
```
BUGSNAG["params_filters"] = ("key", "token")
````

![image](https://user-images.githubusercontent.com/1547881/163600543-aad3385b-49cd-4a77-bca9-cfe01b6a851a.png)

<details><summary>Error messages:</summary>

```

```
</details>

**Comment by luke-belton** [comment]
Hi @justmobilize - thanks for raising this with us. We're going to look at a fix for this and will keep you updated on this thread 👍

**Comment by justmobilize** [comment]
Awesome!

**Comment by yousif-bugsnag** [comment]
This is now fixed in [v4.2.1](https://github.com/bugsnag/bugsnag-python/releases/tag/v4.2.1) 🎉
